src/test/scala/TypeCheckerTest.scala

```diff
@@ -596,6 +596,15 @@ class TypeCheckerTest extends AnyFunSuite {
       mynonEmptyTypechecker.typeof(CallHighOrderExp(IntegerExp(1), List(StringExp("i"))), Map())
     }
   }
+  test("testing MethodExp") {
+
+  }
+  test("MethodExp throws Class is not in string format"){
```

Comment:
I noticed I might have to change method exp cause it should be varexp not stringexp